src/maze.py

```diff
@@ -1,31 +1,32 @@
 from graph import Graph
 
 class Maze:
-    def __init__(self, size):
-        self.size = size
+    def __init__(self, size_n, size_m):
```

Comment:
good because our goal is to make rectangles

Reply:
I agree!